.github/common_edge_cases.md

```diff
@@ -8,3 +8,4 @@ Will this also work if
  - The annotation is a compound task/project/tasktype annotation
  - The user is logged out and views a public dataset or annotation
  - User uses dark mode / light mode
+ - There is no local datastore/tracingstore module
```

Comment:
Good that it is here, but how would I test this?

Reply:
I added a link to https://github.com/scalableminds/webknossos/wiki/Set-up-a-standalone-datastore-locally – I think that’s still mostly up to date. I’ll re-check that the instructions there still work soon